setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = unpaddedbase64
-version = 1.1.0
+version = 2.0.0rc1
```

Comment:
I was a bit worried about how requiring Python 3.6 would affect Synapse's 3.5 compatibility. But since this is marked as a pre-release, even though Synapse specifies `unpaddedbase64 >= 1.1.0`, this shouldn't be picked up.

Reply:
Because we specify our Python version dependency, our generated `PKG-INFO` file will contain `Requires-Python: >=3.6`, which will cause pip on Python 3.5 to not consider this version and instead pull 1.1.0

Reply:
Oh, true indeed.